partcad/src/partcad/sentry.py

```
@@ -0,0 +1,135 @@
+def init_sentry(version: str) -> None:
+    if not sentry_sdk.is_initialized() and user_config.get_string("sentry.dsn"):
+        critical_to_ignore = [
+            "action_start: ",
+            "action_end: ",
+        ]
+        debug_to_ignore = [
+            "Starting action",
+            "Finished action",
+        ]
+
+        def before_send(event, hint):
```

Ruff (0.8.2)

Missing return type annotation for private function `before_send`

(ANN202)